### ISSUE TITLE
Default state for data panel share items (or hide)

### ISSUE THREAD

**Issue by pjsier** [EvictionLab/eviction-maps#658]
Users can still scroll to the data panel even if they haven't selected any features if they scroll on the top bar. If they try to click on the download or social share icons, they'll get something like this:

<img width="719" alt="screen shot 2018-02-20 at 10 07 34 am" src="https://user-images.githubusercontent.com/8291663/36435030-1c6945c8-1626-11e8-8a33-eca2c770d33b.png">

Should we hide this div entirely and only allow people to access the footer below it?

**Comment by Lane** [comment]
I can see a case for keeping it, if someone just wants to access the data or share the blank tool. Maybe we can disable the file export button if nothing is selected? Or show a message instructing them to add locations.

**Comment by pjsier** [comment]
That makes sense to me, and we would just have to add tweet for default state on the map

**Comment by Lane** [comment]
@james-minton can we get some text to use if someone goes to share the tool without any selected locations?

**Comment by james-minton** [comment]
"View the landscape of American housing insecurity with the Eviction Lab's map tool. See eviction rates in your community and across the country, compare locations in charts and graphs, download custom reports, and more."

It's a start. Thanks for considering this use case.

**Comment by pjsier** [comment]
@james-minton and I’m guessing the download button should just be disabled? Do we have any designated styles for that, or should we show a tooltip on hover?

**Comment by james-minton** [comment]
I don't know of designated styles for that, but could a tooltip work just as well?

**Comment by pjsier** [comment]
I’ll give it a shot and see what people think

**Comment by james-minton** [comment]
Thanks Pat, looking forward to seeing it. Re: the text -- I can hear Abby's voice in my head (rightly) telling me it's too wordy. Maybe just cutting what's in brackets below could be enough paring-down?

"View the landscape of American housing insecurity with the Eviction Lab's map tool. See eviction rates in your community [and across the country], compare locations in charts and graphs, download custom reports, and more."

**Comment by Lane** [comment]
Thinking about this some more, any value of showing the graph with only "US Average" when they don't have any locations selected?

**Comment by pjsier** [comment]
That might make sense. Would we still allow people to dismiss US average or would it be fixed?
